### PR TITLE
chore(README): re-did README

### PR DESCRIPTION
**Pull request by Manason**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
+Provides an API to store and retrieve vehicles from various locations around the map. Also includes an optional built-in UI
 
-![image](https://user-images.githubusercontent.com/82112471/149678987-02ec660f-76c9-4414-af7b-bac284ed58b7.png)
+## Features
 
-![image](https://user-images.githubusercontent.com/82112471/149678977-2a574ee9-8ecc-494f-a845-e17281a74594.png)
+### Built-in UI
+Optionally disabled via config. Created using ox_lib
 
+![Garage Menu](example-menu.png)
```
Comment (solareon) [inline README.md:10]: These should be in the .github folder so we don't have to remove them from release versions.

**Review by solareon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Manason) [reply README.md:10]: done

File: README.md [hunk 2]
```diff
@@ -1,47 +1,40 @@
...
-* Park owned cars in public garages.
-* You can only parks vehicles that you own in public garages.
+## Features
 
-![image](https://user-images.githubusercontent.com/82112471/149678987-02ec660f-76c9-4414-af7b-bac284ed58b7.png)
+### Built-in UI
+Optionally disabled via config. Created using ox_lib
 
-![image](https://user-images.githubusercontent.com/82112471/149678977-2a574ee9-8ecc-494f-a845-e17281a74594.png)
+![Garage Menu](example-menu.png)
```
Comment (solareon) [inline README.md:9]:
```suggestion
![Garage Menu](.github/example-menu.png)
```

File: README.md [hunk 3]
```diff
@@ -1,47 +1,40 @@
...
 
+Configurable Markers Denote the pickup and drop off points
+![Markers](example-markers.png)
```
Comment (solareon) [inline README.md:12]:
```suggestion
![Markers](.github/example-markers.png)
```

**Review by solareon — APPROVED** [review]

**Pull request merged**